src/ProjectDependencies/ProjectDependencies.csproj

```diff
@@ -0,0 +1,73 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+	<PropertyGroup>
+		<TargetFrameworks>net7.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net7.0-windows10.0.19041.0</TargetFrameworks>
+		<!-- Uncomment to also build the tizen app. You will need to install tizen by following this: https://github.com/Samsung/Tizen.NET -->
```

Comment:
needed?

Reply:
probably not 🤷

This is all set up by MAUI for me